lib/db.js

```diff
@@ -211,8 +220,29 @@ module.exports = (
   }
 
   DB.prototype.sessions = function (uid) {
-    log.trace({ op: 'DB.sessions', uid: uid })
-    return this.pool.get('/account/' + uid + '/sessions')
+    const promises = [
+      this.pool.get('/account/' + uid + '/sessions')
+    ]
+
+    if (this.redis) {
+      promises.push(this.redis.getAsync(uid))
+    }
+
+    log.trace({ op: 'sessions', uid: uid })
```

Comment:
AFAIK, the point of the `trace` log level is to log function names as they're entered. So I'd keep `op` as `'DB.sessions'`, because that is the object this function was called on.

And for consistency with the other `trace` calls, I'd keep it as first line in the function.